packages/patternfly-4/src/templates/mdxPF4Template.js

```diff
@@ -46,7 +46,7 @@ const MdxPF4Template = ({ data }) => {
       <SEO title="React" />
       <PageSection variant={PageSectionVariants.light} className="section-border pf-site-background-medium">
         <AutoLinkHeader size="md" is="h1" className="pf4-site-framework-title">React</AutoLinkHeader>
-        <AutoLinkHeader size="4xl" is="h2" className="pf-u-mt-sm pf-u-mb-md">
+        <AutoLinkHeader size="4xl" is="h2" id={`${data.mdx.frontmatter.title.replace(/ /g, '-')}-title`} className="pf-u-mt-sm pf-u-mb-md">
```

Comment:
Same comment here, id would overwrite the generated one. perhaps we can add a new suffix prop and pass title into it?